api/src/controllers/AuthController.js

```diff
@@ -1,17 +1,35 @@
 import passport from 'passport';
+import jsonwebtoken from 'jsonwebtoken';
 import { v4 as uuid } from 'uuid';
 import logger from 'lib/logger';
 import User from 'lib/models/user';
 import OAuthToken from 'lib/models/oAuthToken';
 import { sendResetPasswordToken } from 'lib/helpers/email';
-import { AUTH_JWT_SUCCESS } from 'lib/constants/routes';
+import { AUTH_JWT_SUCCESS, AUTH_JWT_REFRESH } from 'lib/constants/routes';
 import {
   ACCESS_TOKEN_VALIDITY_PERIOD_SEC,
   DEFAULT_PASSPORT_OPTIONS
 } from 'lib/constants/auth';
-import { createOrgJWT, createUserJWT } from 'api/auth/jwt';
+import Unauthorized from 'lib/errors/Unauthorised';
+import { createOrgJWT, createOrgRefreshJWT, createUserJWT, createUserRefreshJWT } from 'api/auth/jwt';
 import { AUTH_FAILURE } from 'api/auth/utils';
 
+const buildRefreshCookieOption = protocol => {
+
+  const c = {
+    path: `/api${AUTH_JWT_REFRESH}`,
+    expires: new Date(Date.now() + 7 * 24 * 60 * 60 * 1000),
+    maxAge: 7 * 24 * 60 * 60 * 1000,
```

Comment:
Looks like `7 * 24 * 60 * 60 * 1000` could be a constant, perhaps we call it `refreshExpiration`. What do you think @h-kanazawa?

Reply:
Can we make it an env value to with that as the default?

Reply:
Would you ever consider changing it? I'd rather avoid having too many env values if possible.

Reply:
I guess the refresh is less important, just thinking for QA but they shouldn't need to fiddle with this number

Reply:
Ah right yeah, can understand the need for it in QA. Your call on whether this is required @ht2.

Reply:
I add 3 env variables for QA

- `REFRESH_TOKEN_EXPIRATION_MSEC` (for cookie's expires and maxAge) e.g. "3600000"
- `REFRESH_TOKEN_EXPIRATION`(for jwt exp) e.g. "7d" This value is expected to be same length with `REFRESH_TOKEN_EXPIRATION_MSEC`
- `ACCESS_TOKEN_EXPIRATION` (for jwt exp) e.g. "1h"

Reply:
Okay thanks Hota 👍